### ISSUE TITLE
Settings tree view does not allow for keyboard navigation by category by expanding or collapsing levels

### ISSUE THREAD

**Issue by graham-hardy** [microsoft/vscode#204874]
Type: <b>Bug</b>

Tree views in Windows most commonly allow for items containing other items to be expanded with RightArrow and collapsed with LeftArrow. That is how many other tree views in VS Code work. For example, when navigating the Explorer tree view accessed with Ctrl+Shift+E, you can expand or collapse a folder which contains subfolders and files with RightArrow and LeftArrow. While keyboard focus is on an item that contains other items, you can quickly collapse it with LeftArrow and pass to the next item at the same level with DownArrow.

I would have expected the settings tree view to work this way. It contains several hundred items. Instead, there is no way of collapsing a category; RightArrow and LeftArrow appear to do nothing while in that tree view. You can search for a setting if you know the desired setting or category, but the current behavior affects discoverability of settings.

VS Code version: Code 1.86.1 (31c37ee8f63491495ac49e43b8544550fbae4533, 2024-02-07T09:08:20.941Z)
OS version: Windows_NT x64 10.0.23615
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-10850K CPU @ 3.60GHz (20 x 3600)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|63.83GB (26.32GB free)|
|Process Argv|--crash-reporter-id dea7aed7-8b71-47ae-80fe-be4fcd9ba78a|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (11)</summary>

Extension|Author (truncated)|Version
---|---|---
csdevkit|ms-|1.3.10
csharp|ms-|2.15.30
vscode-dotnet-runtime|ms-|2.0.1
vscodeintellicode-csharp|ms-|0.1.26
remote-wsl|ms-|0.86.0
cmake-tools|ms-|1.16.32
cpptools|ms-|1.19.2
cpptools-extension-pack|ms-|1.3.0
rust-analyzer|rus|0.3.1831
cmake|twx|0.0.17
vscode-lldb|vad|1.10.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscorecescf:30445987
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:30899288
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
h48ei257:30961340
pythontbext0:30879054
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
pydisww2:30959801
3ef8e399:30949928

```

</details>

**Comment by rzhao271** [comment]
I am unable to reproduce this issue on the latest [Insiders](https://code.visualstudio.com/insiders/).

@graham-hardy if you're still seeing this issue on the latest VS Code, could you upload a GIF or video demonstrating the issue with screencast mode on? You can turn on screencast mode in VS Code by opening the Command Palette with Ctrl+Shift+P and running the "Developer: Toggle Screencast Mode" command.
